File: AOP_activities/src/main/java/metier/Account.java
package metier;

public class Account {
    private Long code;
    private double balance;

    public Long getCode() {
        return code;
    }

    @Override
    public String toString() {
        return "Account{" +
                "code=" + code +
                ", balance=" + balance +
                '}';
    }

    public Account() {
    }

    public void setCode(Long code) {
        this.code = code;
    }

    public double getBalance() {
        return balance;
    }

    public void setBalance(double balance) {
        this.balance = balance;
    }

    public Account(Long code, double balance) {
        this.code = code;
        this.balance = balance;
    }
}

File: FirstApp/Entities/Account.cs
namespace FIRSTAPP;

class Account{
    private string id {set; get; }
    private double balance {set; get; }

    public Account(string id, double balance ){
        this.id=id;
        this.balance=balance;
    }

    public void tostring(){
        Console.WriteLine(id);
        Console.WriteLine(balance);
    }
}